site/content/docs/0.0/helpers/stretched-link.md

```diff
@@ -37,18 +37,18 @@ Most custom components do not have `position: relative` by default, so we need t
 </div>
 {{< /example >}}
 
-<!--{{< example >}}
-<div class="row g-0 bg-body-secondary position-relative">
+{{< example >}}
+<div class="row g-none bg-body-secondary position-relative">
```

Comment:
There's a rendering issue here where the negative margin of `.row` is an issue:

![Screenshot 2024-10-23 at 13 16 32](https://github.com/user-attachments/assets/1f120b50-cfa9-4c34-b35e-b4702e283130)

Reply:
Basically, for `.container-fluid` in the documentation, I find it weird too, it should take all the available space:

![Screenshot 2024-10-24 at 06 59 42](https://github.com/user-attachments/assets/234b5fd2-4241-48d6-a300-d3d6605b3d10)

Reply:
Fixed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2744/commits/8042e91b622e1726bb483187c9f7d26c9c3df745